qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/parser/expression/scanner/ExpressionScanner.java

```diff
@@ -107,7 +111,7 @@ protected TokenType internalScan() {
 				bracket++;
 				return finishToken(offset, TokenType.OpenBracket);
 			}
-			stream.advanceUntilChar('(', ')', '"', '\'');
+			stream.advanceUntilChar(MultiLineStream.QUOTE_OR_PAREN);

```

Comment:
I have in my mind to provide a commons parser library and MultiLineStream must be not linked to Qute or other languages. Please define those constant in the scanner implementation.